elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -58,7 +59,7 @@
     private final Map<String, FilterExpression> expressionsByType;
 
     /* Used to filter across heterogeneous types during the first load */
-    private FilterExpression globalFilterExpression;
+    @Setter @Getter private FilterExpression globalFilterExpression;
```

Comment:
I don't think this should have a setter.  There is already a method to get the globalFilterExpression.